python/tests/test_client_integration.py

```
@@ -0,0 +1,162 @@
+    try:
+        client.put("branch-key", b"value")
+        assert client.get("branch-key") == b"value"
+    finally:
+        client.close()
+
+
+def test_bad_ini_path_raises():
+    with pytest.raises(EloqStoreError):
+        Client(
+            Options(
+                options_path="/tmp/does-not-exist-eloqstore.ini",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/does-not-exist-eloqstore.ini"

(S108)